dataflow/src/main/java/org/checkerframework/dataflow/cfg/CFGBuilder.java

```diff
@@ -1716,6 +1716,15 @@ public PhaseOneResult process(CompilationUnitTree root, UnderlyingAST underlying
             return process(bodyPath, underlyingAST);
         }
 
+        /**
+         * Perform any actions required when CFG translation creates a new Tree that is not part of
+         * the original AST, with the corresponding tree path being specified.
+         *
+         * @param tree the newly created Tree
+         * @param path the path for the newly created tree
+         */
+        public void handleArtificialTree(Tree tree, TreePath path) {}
```

Comment:
As this method isn't needed any more, please remove it and inline the single call.